src/test/java/core/basesyntax/FruitTransactionTests.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax;
+
+import core.basesyntax.db.FruitShopStorage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.util.Operation;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class FruitTransactionTests {
```

Comment:
Tests for models classes are also rarely required at all.
You use FruitTransaction massively during handlers testing so model is autocovered during services testing